packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js

```diff
@@ -237,9 +243,10 @@ export default class QuanticNumericFacet extends LightningElement {
     return (
       this.state?.values
         .filter((value) => value.numberOfResults || value.state === 'selected')
-        .map((value) => {
+        .map((value, index) => {
           return {
             ...value,
+            index: index,
```

Comment:
Instead of using the index, I'd suggest to compute a key using the whole range.

    key: `${value.start}..${value.end}`